com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.LookDev.cs

```diff
@@ -242,6 +242,10 @@ void IDataProvider.GetShadowMask(ref RenderTexture output, StageRuntimeInterface
             data.additionalCameraData.clearColorMode = oldClearMode;
         }
 
+        /// <summary>
+        /// Callback called to do any necessary cleanup.
```

Comment:
The callback HDRP raises to process any necessary cleanup.

Reply:
Actually here it's the HDRP implementation of the callback raised by the look dev.

Reply:
Ah okay, how about:
The HDRP implementation for the callback that the look dev raises to process any necessary cleanup.